backend/domain/Player.js

```diff
@@ -4,7 +4,7 @@ class Player {
         this.roomID = roomID;
         this.nickname = nickname;
         this.role = role;
-        this.isAlive = true;
+        this.isAlive = INTIAL_IS_ALIVE;
```

Comment:
This might just be me being dumb, but I can't seem to find a declaration anywhere for `INITIAL_IS_ALIVE`

Also just noticed INITIAL is spelt wrong lol

Reply:
Fixed as you were making the review. Might be good in the future to have `"use strict"` in project files to catch problems like this